createJson.groovy

```diff
@@ -175,6 +175,33 @@ class Generator {
         }
     }
 
+    // for each month, counts the number of JVM versions in use (using strict filtering to ignore weird/local JVM version names)
+    def generateJvmJson() {
+        final def JVM_VERSIONS = ["1.5", "1.6", "1.7", "1.8", "1.9"]
```

Comment:
time to adjust this...

Reply:
@abayer hasn't yet run January's stats, so you could provide a patch.

Reply:
https://github.com/jenkins-infra/infra-statistics/pull/42

Reply:
OMG, answer is always 42 right?